packages/editor-ui/src/router.ts

```diff
@@ -755,12 +755,15 @@ export const routes = [
 	},
 ] as Array<RouteRecordRaw & RouteConfig>;
 
-function withReadonlyMeta(route: RouteRecordRaw & RouteConfig) {
+function withCanvasReadOnlyMeta(route: RouteRecordRaw) {
 	if (!route.meta) {
 		route.meta = {};
 	}
+	route.meta.readOnlyCanvas = !EDITABLE_CANVAS_VIEWS.includes((route?.name ?? '') as VIEWS);
```

Comment:
Suggestion - We could also set the type directly on EDITABLE_CANVAS_VIEWS `VIEWS` in order to avoid type coercion here